pegasus/src/homepage.rb

```diff
@@ -555,10 +555,11 @@ def self.show_courses_banner(request)
 
   def self.get_dance_stars
     stars = [
-      "Katy Perry", "Lil Nas X (ft. Billy Ray Cyrus)", "Jonas Brothers", "Panic! At The Disco",
-      "Shawn Mendes", "Nicki Minaj", "KIDZ BOP", "Pedro Capó", "Francesco Gabbani", "Sia",
-      "A-ha", "Ariana Grande", "Avicii and Aloe Blacc", "Calvin Harris",
-      "Carly Rae Jepsen", "Ciara", "Coldplay", "Ed Sheeran", "Imagine Dragons",
+      "Beyoncé", "Harry Styles", "Lizzo", "Post Malone", "Disney\'s \"Encanto\"", "Nicky Youre",
```

Comment:
Checking that the metadata string `Featuring Beyoncé, Harry Styles, Lizzo, Lil Nas X, Selena Gomez, music from Disney's \"Encanto,\" and more!` and this array do *not* need to match the order of the artists shown in the photo (Beyoncé, Lil Nas X, Harry Styles, Nicky Youre, Nikki Minaj, and Encanto)?

Reply:
Good question, but nope it doesn't! Sam wrote this copy for the description (and we weren't able to get a photo of Beyoncé 🥲).

Reply:
Sounds good! Sad news re: Beyoncé 💃